infra/k8s/sidecar.yaml

```diff
@@ -28,8 +28,7 @@ spec:
         - sh
         - -ac
         - >
-          sleep 20 &&
-          export GW=$(ip route | grep cni0 | awk '{print $7}') &&
+          while [ "$GW" = "" ]; do export GW=$(ip route | grep cni0 | awk '{print $7}'); echo "Got GW: $GW"; sleep 5; done;
           echo $GW &&
           ip route &&
           ip route add 100.64.0.0/16 via $GW &&
```

Comment:
We need the `|| true` so that we can run this multiple times on the same host. On the second and following runs, this route already exists and `ip route` complains.